src/parse/mark.js

```diff
@@ -1,10 +1,17 @@
 var dl = require('datalib'),
     parseProperties = require('./properties');
 
-function parseMark(model, mark) {
+function parseMark(model, mark/*, [isMark]*/) {
   var props = mark.properties,
       group = mark.marks,
-      config = model._config;
+      config = model._config,
+      arglen = arguments.length,
+      argidx = 2,
+      isMark = false;
+
+  if (arglen > argidx && dl.isBoolean(arguments[arglen - 1])) {
+    isMark = true;
```

Comment:
Hi @emilygu, if I follow, `isMark` is there so that we don't set any defaults for axis and legend items? If so, we'll want to make sure we're passing `true` in for the recursive call to `parseMark` in this file (line 59).